### PR TITLE
Solves kata N11

### PR DESCRIPTION
**Pull request by LukaEli**
Hello
I've updated GET /api/articles which filters the articles by the topic value specified in the query

### CODE REVIEW

File: __tests__/app.test.js [hunk 1]
```diff
@@ -120,6 +122,16 @@ describe("GET /api/articles", () => {
...
+  test("GET:200 Responds with articles filtered by topic", () => {
+    return request(app)
+      .get(`/api/articles?topic=mitch`)
+      .expect(200)
+      .then(({ body }) => {
+        body.forEach((article) => {
+          expect(article.topic).toBe("mitch");
+        });
+      });
+  });
```
Comment (nc-nick-jones) [inline __tests__/app.test.js:134]: how about error test for topic that doesn't exist?   And also test to expect status 200 and empty array returned when topic exists in the db but has no articles yet? 
This behaviour will require you to check a topic exists (perhaps use the fetchTopics endpoint you made at the start, or a checkIfExists function if you've made one already.)

**Review by nc-nick-jones — COMMENTED** [review]
nice work. a bit more needed but close!   see comment for details.
